TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -474,5 +474,129 @@ private SymbolReference CreateTableOrAliasName(CodeElementsParser.TableOrAliasNa
                 new SymbolType[] { SymbolType.SqlIdentifier, SymbolType.SqlIdentifier });
             return aliasName;
         }
+
+        public SetAssignmentStatement CreateSetAssignmentStatement(CodeElementsParser.SetAssignmentStatementContext context)
+        {
+            IList<Assignment> assignments = (from assignment in context.assignmentClause() where assignment != null select CreateAssignmentClause(assignment) into assignmentClause where assignmentClause != null select assignmentClause).ToList();
```

Comment:
This expression is hard to read.
- is the first filter (`where assignment != null`) required ?
- This can be rewritten using method syntax instead of query syntax `context.assignmentClause().Select(CreateAssignmentClause).Where(a => a != null).ToList()`